iroh-dns-server/Cargo.toml

```diff
@@ -54,7 +55,7 @@ z32 = "1.1.1"
 hickory-resolver = "0.24.0"
 iroh-net = { version = "0.18.0", path = "../iroh-net" }
 iroh-test = { path = "../iroh-test" }
-mainline = "<1.5.0"
+pkarr = { version = "2.0.0", features = ["rand"] }
```

Comment:
I think @nuhvi has published a new version that does not need this anymore.

Reply:
No, you need this in the tests, since you generate a secret key using pkarr